mirrord/config/src/lib.rs

```
@@ -418,6 +418,27 @@ impl LayerConfig {
                 ))?
             }
         }
+
+        if self.feature.copy_target {
```

Let's also return an error if `self.operator` is `false`.